_posts/2024-06-25-create-your-first-python-package-scipy-2024.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: single
+title: "pyOpenSci Workshop: Create Your First Python Package"
+excerpt: "pyOpenSci will be running our `Create Your First Python Package: Make Your Python Code Easier to Share and Use
+` workshop at SciPy 2024. Read on to learn more!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-workshop-scipy-2024.html
+header:
+  overlay_image: images/blog/2024/june/scipy-2024-workshop.png
+  overlay_filter: rgba(20, 13, 36, 0.8)
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## Create Your First Python Package: Make Your Python Code Easier to Share and Use
+
+* **What:** A hands-on workshop, titled: [Create Your First Python Package: Make Your Python Code Easier to Share and Use](https://cfp.scipy.org/2024/talk/QT9GBY/)
+* **Where:** [SciPy 2024](https://www.scipy2024.scipy.org/), Room 316
+* **When:** Tuesday, July 9th, 2024, from 13:30--17:30 Pacific
```

Comment:
Let's add the repository link for the workshop here:

* https://github.com/pyOpenSci/code-to-module-workshop/
* and the demo python package - https://github.com/pyOpenSci/pyosPackage

Reply:
fixed